words/core.asm

```diff
@@ -7302,7 +7300,7 @@ z_um_star:      rts
 
 
 
-; ## UNLOOP ( -- )(R: n1 n2 n3 ---) "Drop loop control from Return stack"
+; ## UNLOOP ( -- )(R: n1 n2 n3 ---) "Drop current loop control block"
```

Comment:
You can remove the return stack effects on this one too, can't you?

Reply:
good point, there are a few like that I just cleaned up